### ISSUE TITLE
Do you will have a version for mac?

### ISSUE THREAD

**Issue by ghost** [harmonoid/harmonoid#361]
Hello Alex,

Do you will have a version for mac?

Thank you very much for such a wonderful application.

Best regards,

**Comment by alexmercerind** [comment]
Best I can say is that I will try.
Currently, I support the platforms that I personally use on daily basis. I want things to be correctly working & not half baked. macOS will require configuration & some additional work mainly related to configuring dependencies like libmpv etc.

The UI & design principle of apps (one that users prefer) on macOS also seems to be different. I don't want to just support macOS because I want macOS users to use my app. If a software is good, people will automatically find the experience of using it engaging & continue to use it in a long run.

Thanks for the love & support!

**Comment by rafo** [comment]
Just my 2 cents: I would love to have this app on macOS. Looking on the screenshots and the general UI/UX: I would not bother about _compatibility_ and _macOS design principles_.

Configuration: Supporting Linux like (XDG_CONFIG_HOME) a.k.a. ~/.config/harmonoid/ on macOS with config files (of whatever type) is acceptable, even preferred by many people (me being one of them).

Dependencies: Didn't look at the code, but I assume that you don't need to reinvent the wheel here (absolutely no offend from my site!)

Keep up the good work!

**Comment by alexmercerind** [comment]
Yes. We finally have a macOS version, try out the latest update.
